js/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+window.error = (function () {
+  var errorTemplate = document.querySelector('#error').content.querySelector('.error');
+  var errorMessageElement = errorTemplate.cloneNode(true);
+  var errorButton = errorMessageElement.querySelector('.error__button');
+  var errorText = errorMessageElement.querySelector('.error__message');
+
+  return {
+    errorHandler: function (error, errBtnHandler) {
+      errorText.innerText = 'Sending error: ' + error;
+      errorButton.addEventListener('click', errBtnHandler);
+      document.querySelector('body').appendChild(errorMessageElement);
```

Comment:
don't query body every time, cache it in a variable outside the function
`errorElementContainer = document.querySelector('body')`
and then
`errorElementContainer.appendChild(errorMessageElement)`

Reply:
done